rbac/management/workspace/model.py

```diff
@@ -38,3 +38,29 @@ class Workspace(TenantAwareModel):
 
     class Meta:
         ordering = ["name", "modified"]
+
+    def ancestors(self):
+        """Return a list of ancestors for a Workspace instance."""
+        ancestor_ids = [a.uuid for a in self._ancestry_queryset() if a.uuid != self.uuid]
+        ancestors = Workspace.objects.filter(uuid__in=ancestor_ids)
```

Comment:
This query would help so we don't have to do extra query, and get rid off that only() from the serializer:
`        return Workspace.objects.raw(
            """
            WITH RECURSIVE ancestors AS
              (SELECT name, uuid, parent_id
               FROM management_workspace
               WHERE uuid = %s
               UNION SELECT w.name, w.uuid, w.parent_id
               FROM management_workspace w
               JOIN ancestors a ON w.uuid = a.parent_id)
            SELECT uuid AS uuid,
                   uuid AS id,
                   name,
                   parent_id
            FROM ancestors
            WHERE uuid != %s
        """,
            [self.uuid, self.uuid],
        )
`

Reply:
Hey @astrozzc! I'd considered similar as well, but wanted to leave the base method for returning `ancestors` actually returning the full object/queryset in the event that we need those objects' values. The `only()` call should modify the queryset before the query is executed since what comes out of here is the queryset. Thoughts?

Reply:
Got it. Just worried a bit about the extra query, seems redundant. But sounds good to me